dialogTemplate/en.json

```diff
@@ -868,7 +868,7 @@
 					]
 				},
 				{
-					"value": "MRU",
+					"value": "MRO",
```

Comment:
according to wikipedia MRU was correct:
https://en.wikipedia.org/wiki/Mauritanian_ouguiya

Reply:
Ok, my error. I modify.